File: BASIC_CODING/LARGEST_AMONG_2/solution.java
public class Main {
    /**
     * Find and print the maximum between two numbers.
     *
     * @param a The first number.
     * @param b The second number.
     */
    public static void findMax(int a, int b) {
        if (a > b) {
            System.out.println("The maximum number is: " + a);
        } else {
            System.out.println("The maximum number is: " + b);
        }
    }

    public static void main(String[] args) {
        int num1 = 7;
        int num2 = 12;

        System.out.println("Finding the maximum between " + num1 + " and " + num2 + ":");
        findMax(num1, num2);
    }
}

File: BASIC_CODING/LARGEST_AMONG_2/solution.js
/**
 * Find and print the maximum between two numbers.
 *
 * @param {number} a - The first number.
 * @param {number} b - The second number.
 */
function findMax(a, b) {
	if (a > b) {
		console.log(`The maximum number is: ${a}`);
	} else {
		console.log(`The maximum number is: ${b}`);
	}
}

// Example usage:
let num1 = 7;
let num2 = 12;

console.log(`Finding the maximum between ${num1} and ${num2}:`);
findMax(num1, num2);
